challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ Relative.   It has a slash in front of it so there are more directories in front of it.
```

Comment:
Nope, it's an Absolute path! How do we know?
An Absolute path is a path to a resource that doesn't depend on the user's location; whether you're in `/`, `~`, or `~/Users`, the path above will _always_ point you to the same place - the Desktop for user `blah_blah`. 

 A Relative path, in contrast, is a path that's defined by the user's location - e.g. `some_directory_in_the_pwd/some_file`. Depending on where the user is in the file system, the path above will actually point to different locations within the filesystem.